hydrus/data/crud.py

```diff
@@ -410,30 +346,18 @@ def delete_multiple(
 
     """
     id_ = id_.split(',')
-    try:
-        rdf_class = session.query(RDFClass).filter(
-            RDFClass.name == type_).one()
-    except NoResultFound:
-        raise ClassNotFound(type_=type_)
-
     instances = list()
     data_III = list()
     data_IAC = list()
     data_IIT = list()
 
     for index in id_:
-        try:
-            instance = session.query(Instance).filter(
-                Instance.id == index and type_ == rdf_class.id).one()
-            instances.append(instance)
-        except NoResultFound:
-            raise InstanceNotFound(type_=rdf_class.name, id_=index)
-        data_IIT += session.query(triples).filter(
-            triples.GraphIIT.subject == index).all()
-        data_IAC += session.query(triples).filter(
-            triples.GraphIAC.subject == index).all()
-        data_III += session.query(triples).filter(
-            triples.GraphIII.subject == index).all()
+        instance = get_instance_before_delete(session, index, type_)
+        instances.append(instance)
+        data_IAC_index, data_III_index, data_IIT_index = get_data_iac_iii_iit(session, index)
+        data_IIT += data_IAC_index
+        data_IAC += data_IAC_index
+        data_III += data_III_index
 
     data = data_III + data_IIT + data_IAC
```

Comment:
Same here, these checks are already in delete. Maybe we can reuse the function with a few modifications.

Reply:
Yes, I will do this